data/src/main/java/com/carsticaadrian/data/features/news/local/ArticleEntity.java

```diff
@@ -0,0 +1,31 @@
+package com.carsticaadrian.data.features.news.local;
+
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+@Entity(tableName = "articles")
+public class ArticleEntity {
+
+    @PrimaryKey(autoGenerate = true)
+    @Nullable
+    public int id;
+
+    public String imageUrl;
+
+    public String title;
+
+    public String content;
+
+    public String description;
```

Comment:
let's make all the fields final (besides the id); immutable classes are less prone to bugs